src/kvstore/kvstore_local.h

```diff
@@ -135,6 +135,13 @@ class KVStoreLocal : public KVStore {
     PullRowSparseImpl(keys, val_rowids, priority);
   }
 
+  void SetCompress(const std::string& compress, const float neg_threshold,
```

Comment:
nit: Does the name "SetCompression()" make more sense?

Reply:
Hmm yes, that sounds better. Will change the names